mythril/mythril.py

```diff
@@ -481,23 +481,31 @@ def fire_lasers(
 
         all_issues = []
         for contract in contracts or self.contracts:
-            sym = SymExecWrapper(
-                contract,
-                address,
-                strategy,
-                dynloader=DynLoader(
-                    self.eth,
-                    storage_loading=self.onchain_storage_access,
-                    contract_loading=self.dynld,
-                ),
-                max_depth=max_depth,
-                execution_timeout=execution_timeout,
-                create_timeout=create_timeout,
-                transaction_count=transaction_count,
-                modules=modules,
-            )
-
-            issues = fire_lasers(sym, modules)
+            try:
+                sym = SymExecWrapper(
+                    contract,
+                    address,
+                    strategy,
+                    dynloader=DynLoader(
+                        self.eth,
+                        storage_loading=self.onchain_storage_access,
+                        contract_loading=self.dynld,
+                    ),
+                    max_depth=max_depth,
+                    execution_timeout=execution_timeout,
+                    create_timeout=create_timeout,
+                    transaction_count=transaction_count,
+                    modules=modules,
+                )
+                issues = fire_lasers(sym, modules)
+            except KeyboardInterrupt:
+                log.critical("Keyboard Interrupt")
```

Comment:
Is this critical? :rofl:

Reply:
Possibly. It does kind of kill the program. If not critical, what level do you think it should be?